arm-sql/2014-04-01/swagger/capabilities.json

```diff
@@ -0,0 +1,332 @@
+                    ],
+                    "x-ms-enum": {
+                        "name": "MaxSizeUnits"
+                    }
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The status of the maximum size capability.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus"
+                    }
+                }
+            },
+            "description": "Represents the maximum size limits for a database."
+        },
+        "ServiceObjectiveCapability": {
+            "properties": {
+                "name": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The Service Objective name."
+                },
+                "status": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The status of the Service Objective.",
+                    "enum": [
+                        "Visible",
+                        "Available",
+                        "Default"
+                    ],
+                    "x-ms-enum": {
+                        "name": "CapabilityStatus"
+                    }
+                },
+                "performanceLevel": {
+                    "readOnly": true,
+                    "$ref": "#/definitions/PerformanceLevel",
+                    "description": "The performance level of the Service Objective Capability.",
+                    "x-ms-client-flatten": true
+                },
+                "id": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "The unique ID of the Service Objective."
+                },
+                "supportedMaxSizes": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/MaxSizeCapability"
+                    },
+                    "description": "The list of supported maximum database sizes for this Service Objective."
+                }
+            },
+            "description": "Represents the Service Objectives capabilities."
+        },
+        "PerformanceLevel": {            
+            "description": "Represents a possible performance level of a Service Objective Capability.",
+            "properties": {
+                "unit": {
+                    "type": "string",
+                    "description": "Unit type used to measure SLO performance level.",
+                    "enum": [
+                        "DTU"
+                    ],
+                    "x-ms-enum": {
+                        "name": "PerformanceLevelUnit"
```

Comment:
this again sounds like a good candidate for `modelAsString: true` or do you want adding a unit (seems likely from looking at it) to break everyone?